source/extensions/filters/http/oauth2/filter.cc

```diff
@@ -369,15 +375,18 @@ bool OAuth2Filter::canSkipOAuth(Http::RequestHeaderMap& headers) const {
     if (config_->forwardBearerToken() && !validator_->token().empty()) {
       setBearerToken(headers, validator_->token());
     }
+    ENVOY_LOG(debug, "skipping oauth flow due to valid hmac cookie: \n{}", headers);
```

Comment:
Don't think it's a good idea to print all headers. It probably has secrets and would be quite noisy.

Reply:
I have added this as we print in encode headers already. Any way, removed it now.